app/controllers/register_controller.rb

```diff
@@ -52,7 +52,12 @@ def create
     @change = Change.new(register_name: params[:register], payload: payload, user_id: current_user.id)
     @change.save
 
-    RegisterUpdatesMailer.register_update_notification(@change, current_user).deliver_now
+    @change_approvers = current_user.team_members.first.team.users.where.not('team_members.role' => 'basic').reject{ |u| u == current_user }
```

Comment:
Uck, I didn't read this properly the first time.

What you should be doing really is finding the `team` with `params[:register]`, then reject `team_members` where `user_id = current_user.id`. 

The query you are using I don't think will actually always return what you expect because you're using `first`.